### ISSUE TITLE
Remote services: support for EPSG:4326

### ISSUE THREAD

**Issue by cezio** [GeoNode/geonode#3434]
Some services advertise EPSG:4326 but not EPSG:3857 (nor any variation of it), although they can serve with EPSG:3857 quite well, for example:
     
> http://mapy.geoportal.gov.pl/wss/service/img/guest/Administracyjna/MapServer/WMSServer?request=GetCapabilities&version=1.3.0&service=wms
 
returns list of available CRS:     
```
        <CRS>CRS:84</CRS>
        <CRS>EPSG:4326</CRS>
        <CRS>EPSG:2180</CRS>
        <CRS>EPSG:2176</CRS>
        <CRS>EPSG:2177</CRS>
        <CRS>EPSG:2178</CRS>
        <CRS>EPSG:2179</CRS>
```

But it can serve webmercator without an error:

> http://mapy.geoportal.gov.pl/wss/service/img/guest/Administracyjna/MapServer/WMSServer?service=WMS&request=GetMap&layers=Administracyjna&styles=&format=image%2Fpng&transparent=true&version=1.1.1&height=256&width=256&srs=EPSG%3A3857&bbox=1878516.407136492,6613943.183459732,1917652.1656185023,6653078.941941745

This is quite frustrating when adding remote services, because this causes an error, but there's no message, so user ends up with service added, but no layers from that service.

Possible approaches:
 * blindly add services with EPSG:4326, hoping it will allow pseudomercator
 * check if service allows usable CSR anyway

**Comment by francbartoli** [comment]
@cezio is this closed by #3417 ?

**Comment by cezio** [comment]
@francbartoli from the code itself, it doesn't looks like. I'll check this in spare moment.
